web/src/components/tasks/business-formation/BusinessFormationPaginator.tsx

```diff
@@ -371,19 +373,35 @@ export const BusinessFormationPaginator = (): ReactElement => {
               }}
             />
           </div>
-          <div className="flex flex-column-reverse mobile-lg:flex-row mobile-lg:margin-left-auto width-100">
+          <div className="flex mobile-lg:fac flex-column-reverse mobile-lg:flex-row mobile-lg:margin-left-auto width-100">
+            <div className="flex fac margin-top-1 mobile-lg:margin-top-0 mobile-lg:margin-left-auto mobile-lg:margin-right-105">
+              <UnStyledButton
+                style={"basic"}
+                className={"text-accent-cool-darker fjc mobile-lg:width-100 padding-0"}
+                textBold={true}
+                intercomButton
+                noRightMargin={true}
+                dataTestid={"help-button"}
+              >
+                <ButtonIcon svgFilename="help-circle-blue" sizePx="25px" />
+                {Config.formation.general.helpButtonText}
+              </UnStyledButton>
+            </div>
+
```

Comment:
I know the ticket doesn't specifically call this out, but should this also in the DBA/Nexus flow as well?

Reply:
So it exists for Nexus by virtue of being here but you're right that I missed DBA. I will fix